### DISCUSSION TITLE
I can't use the scroll bar style

### DISCUSSION THREAD

**Discussion by Kukaina** [tauri-apps/tauri#8829] [Q&A]
My system is Windows 11，I wrote this code in index.scss, but it does not take effect. Other codes written in index.scss can take effect.
`html {
  scrollbar-width: thin; /* Firefox */
  scrollbar-color: blue orange; /* Firefox */
}

body::-webkit-scrollbar {
  width: 12px; 
}

body::-webkit-scrollbar-track {
  background: orange; 
}

body::-webkit-scrollbar-thumb {
  background-color: blue; 
  border-radius: 20px; 
  border: 3px solid orange; 
}`
My scrollbar still looks like this
![image](https://github.com/tauri-apps/tauri/assets/149920301/e5544274-11eb-40a9-89c5-1e4f73dde061)

**Comment by Skaidon** [comment]
Same issue here.

**Comment by federico-terzi** [comment]
This worked for me on Windows

```
body {
  scrollbar-color: hotpink blue;
}
```

**Reply by Natural-selection1** [reply]
But if you try all the `::-webkit-scrollbar` related pseudo-classes, you'll find that it doesn't work at all

**Comment by CaseSens** [comment]
The issue stems from this code here:

```css
html {
scrollbar-width: thin; /* Firefox /
scrollbar-color: blue orange; / Firefox */
}
```

Your comment syntax is broken and breaks the rest of your css.

Try removing the Firefox comments and you'll see that it works. I tested it and it worked on my Windows 11 machine.

**Reply by CaseSens** [reply]
Proof:

![image](https://github.com/user-attachments/assets/63e45bee-7368-4de5-b8e3-115e4348cbe3)

**Comment by MichaelCharles** [comment]
```
* {
  scrollbar-width: thin;
  scrollbar-color: blue orange;
}
```

The above isn't working for me on Mac. It'll change the width, but not the color.

**Reply by FabianLars** [reply]
webkit doesn't support `scrollbar-*` styles. you'll need to try it with the `::-webkit-*` pseudo elements ([Mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar)), for example for blue orange this may work (not tried myself since i'm on windows rn)
```css
::-webkit-scrollbar-thumb {
  background: blue;
}

::-webkit-scrollbar {
  background: orange;
}
```

**Reply by MichaelCharles** [reply]
That isn't working for me either.

I had been using

```
::-webkit-scrollbar {
  width: var(--scrollbar-size);
  height: var(--scrollbar-size);
}

::-webkit-scrollbar-track {
  background: hsl(var(--scrollbar-track));
  border-radius: var(--scrollbar-radius);
}

::-webkit-scrollbar-thumb {
  background: hsl(var(--scrollbar-thumb));
  border-radius: var(--scrollbar-radius);
}

::-webkit-scrollbar-thumb:hover {
  background: hsl(var(--scrollbar-thumb) / var(--scrollbar-hover-opacity));
}
```

With the variables properly defined, I think that used to work?

I'm opening up an older Tauri project I haven't touched for months and trying to work with it again. The above is the CSS that I had been using; however, now when I run the app, my scrollbars are just grey. 

I tried removing all of my css and just doing

```
* {
  scrollbar-width: thin;
  scrollbar-color: blue orange;
}

::-webkit-scrollbar-thumb {
  background: blue;
}

::-webkit-scrollbar {
  background: orange;
}
```

for simplicity's sake, but that doesn't fit it either.

**Comment by MichaelCharles** [comment]
I was able to get styled scrollbars working by using [overlayscrollbars](https://www.npmjs.com/package/overlayscrollbars). 

I feel like `::-webkit-scrollbar` should work, and if memory serves, it used to work, but for whatever reason it isn't working anymore. At least not on Mac. For now the `overlayscrollbars` approach will be fine for me though.

**Comment by MichaelCharles** [comment]
I threw together [a quick guide](https://michaelcharl.es/aubrey/en/code/styling-tauri-scrollbars-with-overlayscrollbars) to setting up `overlayscrollbars` with React+Tauri+Tailwind.
